src/browser/ui/dom/HTMLDOMPropertyConfig.js

```diff
@@ -104,6 +104,7 @@ var HTMLDOMPropertyConfig = {
     httpEquiv: null,
     icon: null,
     id: MUST_USE_PROPERTY,
+    inputMode: null,
```

Comment:
@zpao definitely seems to be working locally with `null`. Just double checked with Safari, Chrome and Firefox.

Safari
![image](https://cloud.githubusercontent.com/assets/721323/7692515/9e418b34-fe08-11e4-9927-9a92e13d8eed.png)

Chrome
![image](https://cloud.githubusercontent.com/assets/721323/7692524/ab6d8f10-fe08-11e4-8baa-cceb4dc4d4ca.png)

Firefox
![image](https://cloud.githubusercontent.com/assets/721323/7692539/bddbb76c-fe08-11e4-9199-b494e5be33a4.png)

Are you using `inputmode` instead of `inputMode` possibly?